pyproject.toml

```diff
@@ -27,22 +27,35 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
-    "absl-py",
     "etils[epath]",
     "mujoco",
-    "pytest",
+    "numpy",
     "warp-lang",
 ]
 
+[project.optional-dependencies]
+dev = [
+    "absl-py",
```

Comment:
We ship the tests as part of the dist, so probably we should keep absl as part of the core dependencies.

Obviously no outside user should be doing `from mujoco_warp import smooth_test` but they could... so might as well leave the dep for now.  Anyway it's quite a small dep.

Reply:
done